uit/uit.py

```diff
@@ -287,6 +316,8 @@ def prepare_connect(self, system, login_node, exclude_login_nodes, retry_on_fail
         self._username = self._userinfo["SYSTEMS"][self._system.upper()]["USERNAME"]
         self._uit_url = self._uit_urls[login_node]
         self.connected = True
+        self.scheduler = NODE_TYPES[f"{self.system}"]["scheduler"]
```

Comment:
```suggestion
        self.scheduler = NODE_TYPES[self.system]["scheduler"]
```
This can be reduced to not need an f-string.